app/assets/javascripts/header.js

```diff
@@ -94,4 +68,36 @@ function setHeight() {
   $('#header_space').css({"height": the_height});
 }
 
+function expandButton(event) {
+  hideSmallTopNav();
+  console.log(event.data);
+  if (event.data.value % 2 == 0) {
+    showExpandMe();
+  } else {
+    hideExpandMe();
+  }
+  setHeight();
+  event.data.value++;
+}
+
+function headerMouseLeave() {
+  if ($('#expand_moment').length && $('#expand_moment')[0].classList.contains('display_block')) {
```

Comment:
I had the same thoughts about it but I didn't want to change original code too much prior to writing unit tests so I left this bit unchanged. 

BTW `$('#expand_moment')[0].classList.contains('display_block')` is equivalent to `$('#expand_moment').hasClass('display_block')` which is so much better :smile: